microsoft-edge/webview2/release-notes.md

```diff
@@ -31,10 +31,9 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, go to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
-
 ## 1.0.1072.54
   
-Release Date: January 13, 2022
+Release Date: January 13, 2022  
   
 [NuGet package for WebView2 SDK 1.0.1072.54](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1072.54)  
   
```

Comment:
"requires WebView2 Runtime version 95.1020.30"
3-part #?  Supposed to be 4-part?